plugins/woocommerce-blocks/tests/e2e/tests/local-pickup/local-pickup.merchant.block_theme.spec.ts

```diff
@@ -0,0 +1,292 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import { utilsLocalPickup as utils } from './utils.local-pickup';
+
+test.describe( 'Merchant → Local Pickup Settings', () => {
+	test.beforeEach( async ( { admin, page } ) => {
+		await utils.clearLocations( admin, page );
+		await utils.removeCostForLocalPickup( admin, page );
+		await utils.enableLocalPickup( admin, page );
+	} );
+
+	test( 'Renders without crashing, and settings persist', async ( {
+		admin,
+		page,
+	} ) => {
+		await admin.visitAdminPage(
+			'admin.php',
+			'page=wc-settings&tab=shipping&section=pickup_location'
+		);
+		await page.waitForSelector( '#local-pickup-settings' );
+		expect( page.locator( '#local-pickup-settings' ) ).not.toBeNull();
+		await expect(
+			page.locator( '#inspector-checkbox-control-0' )
+		).toBeChecked();
```

Comment:
```suggestion
		await expect( page.getByLabel( 'Enable local pickup' ) ).toBeChecked();
```

I suggest replacing `waitForSelector` with `getByLabel` to get the element directly by its label. CSS classes and IDs might change over time, whereas texts are more resilient. Apart from that, the tests are closer to what a real user would do. In pdToLP-10h-p2, @opr shared best practices when migrating tests from Puppeteer to Playwright. One thing, he recommended, was avoiding `waitFor` statements, when possible.

Reply:
I've applied these changes locally and will push them soon.